ArmRobot/Assets/Scripts/DomainRandomization.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DomainRandomization : MonoBehaviour
+{
+    public GameObject table;
+    public GameObject cube;
+    public GameObject robot;
+    float robotMinReach;
+    float robotMaxReach;
+    
+
+
+    public Vector3 scaleObject = new Vector3 (0.1f, 0.1f, 0.1f);
+
+    public int nbMaxDistractorObjects = 8;
+    public float rotationAngle = 10.0f;
+
+    public float yAltitudeTable = 0.813f;
+    public float minimumDistanceWithoutObjects = 0.05f;
+
+    public int nbMaxLights = 3;
+
+    
+    void start()
```

Comment:
The `Start()` method must start with a capital letter, or else it won't be automatically called at runtime.